boto3/data/ec2/2016-11-15/resources-1.json

```diff
@@ -2577,6 +2618,50 @@
           }
         }
       }
+    },
+    "VpnGateway": {
+      "identifiers": [
+        {
+          "name": "VpnGatewayId"
```

Comment:
Some may argue this resource file is full of this exact anti-pattern. 😉 

I have updated to use Id in my new PR and will update all future PRs with this logic. 😄